Source/Scene/Cesium3DTile.js

```diff
@@ -899,13 +910,43 @@ define([
         return new TileOrientedBoundingBox(center, halfAxes);
     }
 
-    function createRegion(region, result) {
-        var rectangleRegion = Rectangle.unpack(region, 0, scratchRectangle);
+    function createBoxFromTransformedRegion(region, transform, initialTransform, result) {
+        var rectangle = Rectangle.unpack(region, 0, scratchRectangle);
+        var minimumHeight = region[4];
```

Comment:
Since we just created an instance of a Rectangle, I think it would be more readable to use `east`, `north`, etc class members rather than the accessor. Here and in `createRegion`.

Reply:
The reason for the `region[4]` and `region[5]` is `Rectangle` doesn't contain min/max height properties.